extensions/wikia/Email/Email.i18n.php

```diff
@@ -8,7 +8,6 @@
 
 	// General errors
 	'emailext-error-restricted-controller' => 'Access to this controller is restricted',
-	'emailext-error-noname' => 'Required username has been left empty',
```

Comment:
There's a few messages inside `EmailController` that are being needlessly i18n'ed, so I started removing them and replacing them with English strings inside the `throw` statements. Didn't want to look at every one, but in general we can start removing these message keys that are meant only for developers to see.